multi-transfer-esdt/src/lib.rs

```diff
@@ -87,14 +95,17 @@ pub trait MultiTransferEsdt:
             }
 
             // emit event before the actual transfer so we don't have to save the tx_nonces as well
-            self.transfer_performed_event(
-                batch_id,
-                eth_tx.from.clone(),
-                eth_tx.to.clone(),
-                eth_tx.token_id.clone(),
-                eth_tx.amount.clone(),
-                eth_tx.tx_nonce,
-            );
+            // emit events only for non-SC destinations
```

Comment:
do we have another event if the destination is a sc ?

Reply:
Added. 
This solves issue 9 from Audit [document](https://docs.google.com/document/d/1-KDYd9DR9vKQnw08mzifbeVJ9TAIiOrvvX3MvWmEjsA/edit?tab=t.0#heading=h.3cq62w697v91).